tests/test_unit_conversion.py

```diff
@@ -748,6 +747,25 @@ def test_convert_same_unit_different_name(self, br_model):
         direction = DataDirectionFlow.INPUT
         assert br_model.convert_variable(variable, unit, direction) == variable
 
+    def test_convert_state_symbols(self, aslanidi_model):
```

Comment:
Is the aslanidi_model a shared fixture? If so, this could break other tests in the future.

Edit: yes, it is session-scoped.

Reply:
I'd prefer to test by extending `test_multiple_odes_1` so the `multiode_model` has `x = 1{mV_per_ms} + ode(sv1, time) * 3{mV_per_ms}` (or similar) that causes the original code to fail.

Reply:
done